tests/integration/externalapi/test_venafiapi.py

```diff
@@ -25,70 +30,108 @@ def with_random_name(func):
     '''
     generate a randomized name for a container
     '''
+
     @functools.wraps(func)
     def wrapper(self, *args, **kwargs):
         name = _random_name(prefix='salt_')
-        return func(self, _random_name(prefix='salt_test_'), *args, **kwargs)
+        return func(self, _random_name(prefix='salt-test-'), *args, **kwargs)
+
     return wrapper
 
 
-@destructiveTest
-@expensiveTest
 class VenafiTest(ShellCase):
     '''
     Test the venafi runner
     '''
 
     @with_random_name
-    def test_gen_key_password(self, name):
-        '''
-        venafi.gen_key
-        '''
-        ret = self.run_run_plus(fun='venafi.gen_key',
-                                minion_id='{0}.test.saltstack.com'.format(name),
-                                dns_name='{0}.test.saltstack.com'.format(name),
-                                zone='Internet',
-                                password='SecretSauce')
-        self.assertEqual(ret['out'][0], '-----BEGIN RSA PRIVATE KEY-----')
-        self.assertEqual(ret['out'][1], 'Proc-Type: 4,ENCRYPTED')
-        self.assertEqual(ret['out'][-1], '-----END RSA PRIVATE KEY-----')
+    def test_request(self, name):
+        cn = '{0}.example.com'.format(name)
+        print("requesting cn "+cn)
```

Comment:
The linter is complaining on this line. The best fix would be to add relevant information to the assertions or their message. It could be changed to 

    'venafi_certificate not found in `output_value` for {cn}'.format(cn=cn)

For instance.

Reply:
Alternatively you could add `from __future__ import print_function` to the top of the file.